### PR TITLE
Minimum Viable Product

### PR DESCRIPTION
**Pull request by Reggeditt**
In this PR:
- we created a catalogue app with the following options for users:
     - list all books
     - list all music albums
     - list all games
     - list all authors
     - list all genres
     - list all labels
     - add a book
     - add a music album
     - add a game
     - add an author

- added tests for all classes
- added schema mirroring the implemented classes.

click [here](https://drive.google.com/file/d/1U8LVpmkEmn5zLwWKFKxGYYIRdgCJbmuC/view?usp=sharing) for a walkthrough video on the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,158 @@
...
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+- Rspec
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Project Demonstration
+View a video demo of our project [here](https://drive.google.com/file/d/1U8LVpmkEmn5zLwWKFKxGYYIRdgCJbmuC/view?usp=sharing)
```
Comment (Ismailco) [inline README.md]: - [ ] Great work on the video presentation however the video ended strangely while you were talking, kindly consider recording a new one in which all the students in the group have a chance to talk and present a piece of code you're proud of making, and please try to keep it 5 minutes max

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  last_name varchar(50)
+);
+
+-- create games table
+CREATE TABLE IF NOT EXISTS games(
+  id serial not null primary key,
+  multiplayer boolean,
+  last_played date,
+  publish_date date
+
```
Comment (Ismailco) [inline database/schema.sql]: - [ ] Here you forgot to close the `(` at the end of the query, kindly make sure to fix this so the code doesn't produce an error

```suggestion
CREATE TABLE IF NOT EXISTS games(
  id serial not null primary key,
  multiplayer boolean,
  last_played date,
  publish_date date
)

```

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
- [x] Great job with the app so far I can save the data to a json file however when I remove the files the app crashes, kindly make sure to fix this so it works properly

Screenshot_20230812_121900


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by fezzopro** [comment]
@Ismailco 
Thank you for your feedback. We're happy that we have implemented all your fedback

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED ✔️ 
Hi @deniodev, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Alt Text
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**